tests/CMakeLists.txt

```diff
@@ -1,5 +1,6 @@
 # GTest setup.
 add_subdirectory(gtest)
+
 include_directories("${gtest_SOURCE_DIR}/include")
```

Comment:
Nit: This line is not added by this PR, but it seems redundant. `"${gtest_SOURCE_DIR}/include"` is included in the `CMakeLists.txt` under `gtest` subdirectory.